backend/hitas/views/utils/pdf.py

```diff
@@ -0,0 +1,17 @@
+from io import BytesIO
+from typing import Any
+
+from django.http import HttpResponse
+from django.template.loader import get_template
+from rest_framework import exceptions
+from xhtml2pdf import pisa
+
+
+def render_to_pdf(template: str, context: dict[str, Any]) -> HttpResponse:
```

Comment:
small nitpick: IMO this does a bit more than renders to pdf, it creates a HttpResponse. I would make this helper return the `result.getvalue()` and create the HttpResponse in the calling view. Other option is to rename this to `pdf_response` or similar